### PR TITLE
Cat 1343 replace cat codes d i j r t q with open closed on the supervisor page

### PR DESCRIPTION
**Pull request by Olusola-Opencast**
CAT-1343 replace d, i, j, t and q with open/closed on supervisor page, offender history section, tasklist page and stats page where required.

### CODE REVIEW

File: server/views/formPages/approvedView.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
 {% endblock %}
 
 {% block content %}
+{% set isFemale = user.activeCaseLoad.female %}
```
Comment (kberec) [inline server/views/formPages/approvedView.html:17]: Is it needed?

File: server/views/formPages/recat/approvedView.html [hunk 2]
```diff
@@ -11,6 +11,7 @@
 
 {% endblock %}
 {% block content %}
+{% set isFemale = user.activeCaseLoad.female %}
```
Comment (kberec) [inline server/views/formPages/recat/approvedView.html:14]: Is it needed?

File: server/views/formPages/supervisor/recatReview.html [hunk 3]
```diff
@@ -7,6 +7,7 @@
 
 {% set greenButtonText = "Submit" %}
 {% set pageTitle = "Approve category" %}
+{% set isFemale = user.activeCaseLoad.female %}
```
Comment (kberec) [inline server/views/formPages/supervisor/recatReview.html:10]: Same here

File: server/views/formPages/supervisor/review.html [hunk 4]
```diff
@@ -19,7 +19,7 @@
 {% endblock %}
 
 {% block formItems %}
-
+{% set isFemale = user.activeCaseLoad.female %}
```
Comment (kberec) [inline server/views/formPages/supervisor/review.html:22]: Same question here

File: server/views/partials/offenderBanner.html [hunk 5]
```diff
@@ -24,14 +24,13 @@
     dateConverter(data.details.dateOfBirth) }}
   </div>
   <div id="category" class="govuk-grid-column-one-quarter govuk-!-font-weight-bold">
+    <!-- I/J/R/T must show as YOI closed/YOI open/open/closed the men's categories must show as B, C, D -->
```
Comment (kberec) [inline server/views/partials/offenderBanner.html:27]: Leftover comment

File: server/views/formPages/supervisor/recatReview.html (hunk 3 above)
Comment (Olusola-Opencast) [reply server/views/formPages/supervisor/recatReview.html:10]: This is required on line 115

File: server/views/formPages/supervisor/review.html (hunk 4 above)
Comment (Olusola-Opencast) [reply server/views/formPages/supervisor/review.html:22]: This is used further down on line 148 but is in the wrong place.

**Review by Lisaraemoj — COMMENTED** [review]
Looks good to me 👍

**Review by kberec — APPROVED** [review]

**Pull request merged**